Dockerfile

```diff
@@ -0,0 +1,53 @@
+#base image
+FROM ubuntu:20.04
+
+MAINTAINER OriginTrail
+LABEL maintainer="OriginTrail"
+ENV NODE_ENV=testnet
+
+#Install git, nodejs, mysql, python
+RUN apt-get -qq update && apt-get -qq -y install curl
+RUN curl -sL https://deb.nodesource.com/setup_14.x |  bash -
+RUN apt-get -qq update
+RUN apt-get -qq -y install wget apt-transport-https
+RUN apt-get -qq -y install git nodejs
+RUN apt-get -qq -y install mysql-server unzip nano
+RUN apt-get -qq -y install make python
+
+
+
+#supervisor install
+RUN apt-get update && apt install -y -qq supervisor
+RUN mkdir -p /var/log/supervisor
+COPY supervisord.conf /etc/supervisor/conf.d/supervisord.conf
+
+#Install forerver and nodemon
+RUN npm install forever -g
+RUN npm install nodemon -g
+
+#Copyping origintrail_nodeirc to image
+ADD .origintrail_noderc_example /ot-node/current/.origintrail_noderc
+
+WORKDIR /ot-node/current
+
+COPY package*.json ./
+
+#Install nppm
+RUN npm install
+RUN npm ci --only=production
+RUN npm install --save form-data
```

Comment:
If we are missing form-data dependency then we should include it in package.json, no need to install it manually here, just run that command and add updated package.json to the PR. Then it docker file should work without this line